src/Traits/AvailableActionTrait.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Opstalent\CrudBundle\Traits;
+
+use Doctrine\Common\Annotations\AnnotationException;
+use Opstalent\CrudBundle\Exception\UnavailableActionException;
+
+/**
+ * Trait AvailableActionTrait
+ * @author Szymon Kunowski <szymon.kunowski@gmail.com>
+ * @package Opstalent\CrudBundle
+ */
+trait AvailableActionTrait
```

Comment:
`ActionAvailableTrait`

IMO it will be better

Reply:
but why ? 
there is ActionTrait where we operate on Actions and we Have AvailableActionTrait where we operate on AvailableActions
ActionAvailable sounds strage to me.